TestPlaybooks/playbook-GoogleCloudTranslate-Test.yml

```diff
@@ -0,0 +1,361 @@
+      version: -1
+      name: Auto Detect Language - Check Result
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      "yes":
+      - "10"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: GoogleCloudTranslate
+                accessor: TranslateText(val.text=="שלום עולם").translated_text
+                transformers:
+                - operator: toLowerCase
+            iscontext: true
+          right:
+            value:
+              simple: ciao mondo
+      - - operator: isEqualString
+          left:
+            value:
+              simple: GoogleCloudTranslate.TranslateText(val.text=="שלום עולם").detected_language_code
+            iscontext: true
+          right:
+            value:
+              simple: iw
+    view: |-
+      {
+        "position": {
+          "x": 950,
+          "y": 590
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "8":
+    id: "8"
+    taskid: 7d19a10d-a225-4712-8363-818943be1ff0
+    type: regular
+    task:
+      id: 7d19a10d-a225-4712-8363-818943be1ff0
+      version: -1
+      name: Force Source Language
+      description: Translate Text - translate text
+      script: '|||gct-translate-text'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "9"
+    scriptarguments:
+      source:
+        simple: en
+      target:
+        simple: it
+      text:
+        simple: confetti
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 410
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "9":
+    id: "9"
+    taskid: f7fcecb7-3b52-48d1-8007-7209fb1fafa5
+    type: condition
+    task:
+      id: f7fcecb7-3b52-48d1-8007-7209fb1fafa5
+      version: -1
+      name: Force Source - Check Result
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      "yes":
+      - "10"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: GoogleCloudTranslate
+                accessor: TranslateText(val.text=="confetti").translated_text
+                transformers:
+                - operator: toLowerCase
+            iscontext: true
+          right:
+            value:
+              simple: coriandoli
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 590
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "10":
+    id: "10"
+    taskid: 770bad14-37ae-4439-8cb7-e28e6d03f0c7
+    type: regular
+    task:
+      id: 770bad14-37ae-4439-8cb7-e28e6d03f0c7
+      version: -1
+      name: Close Investigation
```

Comment:
don't close the test investigation, instead just finish with the Done section (task No. 2)

Reply:
I have removed the step in the new commit. I implemented it because of https://demisto.pan.dev/docs/testing#close-the-investigation
Is that doc outdated?